src/internal/observable/empty.ts

```diff
@@ -62,7 +62,7 @@ export const EMPTY = new Observable<never>(subscriber => subscriber.complete());
  * the emission of the complete notification.
  * @return An "empty" Observable: emits only the complete
  * notification.
- * @deprecated Deprecated in favor of using {@link index/EMPTY} constant, or {@link scheduled} (e.g. `scheduled([], scheduler)`)
+ * @deprecated Deprecated in favor of using {@link EMPTY} constant, or {@link scheduled} (e.g. `scheduled([], scheduler)`)
```

Comment:
There's a problem with the link when `index/` is removed.

Running the docs locally gives me this:

![image](https://user-images.githubusercontent.com/1096332/70845259-90912300-1e4c-11ea-821a-d7c4b131e3c6.png)

And consequently, the link leads to "Page Not Found" when clicked.